smart-contracts/justfile

```diff
@@ -0,0 +1,72 @@
+orc_config := env_var_or_default('CONFIG', '`pwd`/ci/configs/cosm-orc/ci.yaml')
+test_addrs := env_var_or_default('TEST_ADDRS', `jq -r '.[].address' ci/configs/test_accounts.json | tr '\n' ' '`)
+gas_limit := env_var_or_default('GAS_LIMIT', '10000000')
+
+build:
+	cargo build
+
+test:
+	cargo test
+
+lint:
+	cargo +nightly clippy --all-targets -- -D warnings
+
+gen: build gen-schema
+
+gen-schema:
+	./scripts/schema.sh
+
+integration-test: deploy-local workspace-optimize
```

Comment:
Remove this command we do not use and check all the other ones for similar cases